sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/HashAggregateExec.scala

```diff
@@ -932,12 +932,12 @@ case class HashAggregateExec(
         val functionString = Utils.truncatedString(allAggregateExpressions, "[", ", ", "]")
         val outputString = Utils.truncatedString(output, "[", ", ", "]")
         if (verbose) {
-          s"HashAggregate(keys=$keyString, functions=$functionString, output=$outputString)"
+          s"$nodeName(keys=$keyString, functions=$functionString, output=$outputString)"
```

Comment:
We don't need care about `Exec` suffix since it has been replaced:
https://github.com/apache/spark/blob/299d297e250ca3d46616a97e4256aa9ad6a135e5/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/trees/TreeNode.scala#L420-L424